packages/tools/f-vue-icons/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v2.3.0
+------------------------------
+*January 18, 2020*
+
+### Changed
+- Replace gift and arrow icons to match new designs.
+
```

Comment:
Super minor, can you add a double line break so it's consistent with the other entries please.

Reply:
Done:heavy_check_mark: